__mocks__/patient.mock.ts

```diff
@@ -114,7 +114,7 @@ export const mockPatientAge4 = {
   ],
 };
 
-export const mockPatientAge16 = {
+export const mockPatientAge8 = {
```

Comment:
The underlying patient record is 8 years relative to the static referenced date.